### ISSUE TITLE
NVDA should support unicode Emoticons in both Braille and speech

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5135]
**Reported by winman3000 on 2015-06-02 21:51**
More and more People are using Smartphones and they sent Unicode Emoticons. NVDA cannot announce These emotecons and cannot Show it correctly on the Braille Display either.

**Comment by leonardder** [comment]
Closing as older duplicate of #6523. Note that #6523 deals with speech in particular, but I guess braille is also quite important.
